WalletWasabi.Fluent/Views/Wallets/WalletSettingsView.axaml

```diff
@@ -18,6 +18,11 @@
         <TextBlock Text="PSBT workflow" />
         <ToggleSwitch IsChecked="{Binding PreferPsbtWorkflow}" />
       </DockPanel>
+
+      <DockPanel>
```

Comment:
This should be hidden for WatchOnly wallets.
Add a new property `IsWatchOnly = wallet.KeyManager.IsWatchOnly;`
And you can use it here like `IsVisible="{Binding !IsWatchOnly}"`

Reply:
Done